src/app/modules/verification/verification.routes.ts

```diff
@@ -1,65 +1,74 @@
 import { celebrate, Joi } from 'celebrate'
-import { Express } from 'express'
+import { Router } from 'express'
 
 import verifiedFieldsFactory from './verification.factory'
 
-const mountVfnRoutes = (app: Express): void => {
-  const formatOfId = Joi.string().length(24).hex().required()
-  app.route('/transaction').post(
-    celebrate({
-      body: Joi.object({
-        formId: formatOfId,
-      }),
+const VfnRouter = Router()
+
+const formatOfId = Joi.string().length(24).hex().required()
+
+VfnRouter.post(
+  '/',
+  celebrate({
+    body: Joi.object({
```

Comment:
Since you are already touching this file, do you want to extract these to `verification.rules` instead? Or will that be done in a different PR.

If doing now, suggest using `Segments.BODY` from `celebrate` instead of using `body` key directly

Reply:
I considered it, but I decided against it because I think since the rules are fairly simple, moving them to a separate file is more confusing than clean. e.g. if you want to check which rules apply to which function, you would have to check back and forth between 3 files, which is frustrating. wdyt?